product.json

```diff
@@ -19,7 +19,7 @@
 	"linuxIconName": "com.azuredatastudio.oss",
 	"licenseFileName": "LICENSE.txt",
 	"reportIssueUrl": "https://github.com/Microsoft/azuredatastudio/issues/new?labels=customer%20reported%20issue",
-	"requestFeatureUrl": "https://github.com/Microsoft/azuredatastudio/issues/new?labels=feature-request",
+	"requestFeatureUrl": "https://go.microsoft.com/fwlink/?linkid=2118021",
```

Comment:
I feel like it just makes more sense to use hard links. Its not like we want to change them that often and forward links can be hard to track down. Just seems like an unnecessary hassle to use forward links.

Reply:
So oddly enough I was having problems getting the link opener to open https://github.com/Microsoft/azuredatastudio/issues?q=is%3Aopen+is%3Aissue+label%3AEnhancement+sort%3Areactions-%2B1-desc correctly. It wouldn't escape the %3A correctly and result in a broken link. I didn't really want to dive into trying to fix that for this relatively small issue so just went with fwlinks.

Reply:
The general guidance has been "never" ship hardcode links in client software, in case something happens to the target website.  i.e. it goes offline, gets hacked, changes query string handling, etc.  So the links could be fixed without needing to service client bits.  Though in this case I think the risk of a hard-code link seems fairly minimal, so I'm fine either way.